metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorDatabaseService.java

```diff
@@ -108,6 +110,24 @@ public void delete(@Nonnull final ConnectorContext requestContext, @Nonnull fina
      * {@inheritDoc}.
      */
     @Override
+    public void update(@Nonnull final ConnectorContext context, @Nonnull final DatabaseInfo databaseInfo) {
+        try {
+            this.metacatHiveClient.alterDatabase(databaseInfo.getName().getDatabaseName(),
```

Comment:
You call `getName()` a lot in here. Might be better to save it to a local variable. The compiler probably optimizes this to squash the function call but if we ever change the implementation of `getName` to do some processing under the hood you would replicate executing that functionality multiple times this way. If you put it in a local variable you only have to do it one time. good overall advice